pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -71,7 +71,7 @@ def backfill_output_model_fips(self, fips, t0, final_beds, output_model):
             Backfill dataframe.
         """
         backfill_to_date = date(2020, 3, 3)   # @TODO: Parameterize
-        hospitalization_rate = 0.04           # @TODO: Parameterize
+        hospitalization_rate = 0.02           # @TODO: Parameterize
```

Comment:
Not parametrizing because this is going away today or tomorrow when we swap to inference models.